src/main/java/pl/umk/workshop/javaintroduction/domain/models/User.java

```diff
@@ -0,0 +1,43 @@
+package pl.umk.workshop.javaintroduction.domain.models;
+
+import java.util.Objects;
+
+public class User extends PlatformUser {
```

Comment:
Być może ta klasa może być recordem, może można to pokazać na końcu po zadaniach

Reply:
specjalnie tego nie dałem, bo mam zamiar o tym powiedzieć 😉